### ISSUE TITLE
Convert all on_app_init calls to on_plugin_enable (Trac #25)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#29]
Migrated from http://microfluidics.utoronto.ca/ticket/25
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Convert all on_app_init calls to on_plugin_enable", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T03:26:29", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120220T04:08:44: ryan changed summary from "convert all on_app_init calls to on_plugin_enable" to "Convert all on_app_init calls to on_plugin_enable"

**Comment by cfobel** [comment]
Trac update at 20120403T15:14:07: 
* ryan changed owner from "ryan" to "cfobel"
* ryan changed status from "new" to "assigned"

**Comment by cfobel** [comment]
Trac update at 20120408T06:27:15: 
* christian@fobel.net commented:

> commit bb7ef60e29bb4b13a4ddb167f6695cbc54660fc2
> Author: Christian Fobel <christian@fobel.net>
> Date:   Sun Apr 8 02:23:12 2012 -0400
> 
>     Convert all on_app_init calls to on_plugin_enable
>     
>     Fixes #29.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "assigned" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
